addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,16 @@
+---Models in module 'calendar'---
+---Fields in module 'calendar'---
+calendar     / calendar.alarm           / alarm_type (selection)        : NEW required, selection_keys: ['email', 'notification'], req_default: function, hasdefault
+calendar     / calendar.alarm           / type (selection)              : DEL required, selection_keys: ['email', 'notification'], req_default: function
+# DONE: pre-migration: renamed fields
+
+calendar     / calendar.event           / event_tz (selection)          : NEW selection_keys: function, hasdefault
+# NOTHING TO DO: New field that has default, will be automatically filled
```

Comment:
I don`t know why but I have a null value in an event_tz column after migration.
Do you have the same problem?

Reply:
I will check. But null value in this field means means that `self.env.context.get('tz') or self.user_id.tz` is null 🤔

Reply:
Well, the field is not required, so no serious problem if it has null values.

Reply:
You are right:
self.env.context.get('tz') == None
and
self.user_id == res.user() => self.user_id.tz == False

Reply:
I'm pushing a revision making sure this field is empty for all for preserving v12 behavior.